libdevcrypto/Common.h

```diff
@@ -96,6 +96,21 @@ void encryptSym(Secret const& _k, bytesConstRef _plain, bytes& o_cipher);
 /// Symmetric decryption.
 bool decryptSym(Secret const& _k, bytesConstRef _cipher, bytes& o_plaintext);
 
+/// Encrypt payload using ECIES standard with AES-CTR. TODO: move into class.
+void encryptECIES(Public const& _k, bytesConstRef _plain, bytes& o_cipher);
+	
+/// Decrypt payload using ECIES standard with AES-CTR. TODO: move into class.
+bool decryptECIES(Secret const& _k, bytesConstRef _cipher, bytes& o_plaintext);
+	
+/// Encrypts payload with random IV using AES-CTR. TODO: prefix IV.
+h128 encryptSymNoAuth(Secret const& _k, bytesConstRef _plain, bytes& o_cipher);
+
+/// Encrypts payload with specified IV using AES-CTR TODO: prefix IV.
+h128 encryptSymNoAuth(Secret const& _k, bytesConstRef _plain, bytes& o_cipher, h128 const& _iv);
```

Comment:
In this case a default argument is somewhat dangerous and a ref is needed rather than a copy.